examples/evaluation/rephrase/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "ollama-llama3-2": {"api":"ollama", "model_name": "llama3.2", "parameters": {"temperature": 0}}
```

Comment:
Small thing but in the example notebook you suggest people to pull gemma but then in the example setting you have llama3.2

Reply:
Ignore this, I read it again and it makes sense - i'll add a small comment there